src/files-regular/cat-pull-stream.js

```diff
@@ -21,24 +21,19 @@ module.exports = (common, options) => {
 
     before(async () => { ipfs = await common.setup() })
 
-    before((done) => ipfs.add(fixtures.smallFile.data, done))
+    before(() => ipfs.add(fixtures.smallFile.data))
     after(() => common.teardown())
 
-    it('should return a Pull Stream for a CID', (done) => {
+    it('should return a Pull Stream for a CID', async () => {
       const stream = ipfs.catPullStream(fixtures.smallFile.cid)
 
-      pull(
-        stream,
-        pull.concat((err, data) => {
-          expect(err).to.not.exist()
-          expect(data.length).to.equal(fixtures.smallFile.data.length)
-          expect(data).to.eql(fixtures.smallFile.data.toString())
-          done()
-        })
-      )
+      const data = Buffer.concat(await pullToPromise.any(stream))
```

Comment:
IMO pull-streams pipelines should not be transformed in promises, you lose flexibility and gain 0 value. Pull-streams should either disappear in the future or be left to work as they were created.